driver/razerchromacommon.c

```diff
@@ -755,6 +755,27 @@ struct razer_report razer_chroma_extended_matrix_set_custom_frame2(unsigned char
     return report;
 }
 
+struct razer_report razer_pro_type_matrix_effect_static(unsigned char variable_storage, unsigned char led_id, unsigned char brightness)
+{
+    struct razer_report report = razer_chroma_extended_matrix_effect_base(0x09, variable_storage, led_id, 0x01);
+
+    report.arguments[4] = 0x01;
+    report.arguments[5] = 0x01;
+    report.arguments[6] = brightness;
+    return report;
```

Comment:
I'm not convinced making a custom function for this device is necessary.

The `razer_chroma_extended_matrix_effect_static` function doesn't work for this device? In the pcap I see rgb=0xFFFFFF there except that `report.argument[4] = 0x01` , everything else looks to be the same.

Same with breathing effect.